platform/ios/src/MGLMapView.mm

```diff
@@ -770,6 +773,49 @@ - (void)layoutSubviews
     }
 }
 
+/// Updates `contentInset` to reflect the current window geometry.
+- (void)adjustContentInset
+{
```

Comment:
It [doesn’t](https://github.com/mapbox/mapbox-gl-native/pull/3583/files#diff-20502b6bc02986a4398d0da84b2d4b94R779), for consistency with UIScrollView and `UIViewController.automaticallyAdjustsScrollViewInsets`. (This is already more robust than the UIScrollView behavior; a UIScrollView has to be the first subview.) Regardless, a developer can manually set the `contentInset` of a deeply nested MGLMapView.